still failing
```
===================================================================
ERROR: test_zipped_export_writer_unicode (couchexport.tests.test_writers.ZippedExportWriterTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/couchexport/tests/test_writers.py", line 32, in test_zipped_export_writer_unicode
    self.writer._write_final_result()
  File "corehq/ex-submodules/couchexport/writers.py", line 295, in _write_final_result
    archive.write(path, self._get_archive_filename(name))
  File "corehq/ex-submodules/couchexport/writers.py", line 300, in _get_archive_filename
    return os.path.join(self.archive_basepath, u'{}{}'.format(name, self.table_file_extension))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe3 in position 0: ordinal not in range(128)
----------------------------------------------------------------------

````

--- a/corehq/ex-submodules/couchexport/tests/test_writers.py
+++ b/corehq/ex-submodules/couchexport/tests/test_writers.py
@@ -1,4 +1,5 @@
 # coding: utf-8
+import os
 from codecs import BOM_UTF8
 
 import StringIO
@@ -18,7 +19,7 @@ class ZippedExportWriterTests(SimpleTestCase):
         self.path_mock.get_path.return_value = 'tmp'
 
         self.writer = ZippedExportWriter()
-        self.writer.archive_basepath = 'path'
+        self.writer.archive_basepath = '✓path'
         self.writer.tables = [self.path_mock]
         self.writer.file = Mock()
 
@@ -30,13 +31,19 @@ class ZippedExportWriterTests(SimpleTestCase):
         mock_zip_file = self.MockZipFile.return_value
         self.writer.table_names = {0: u'ひらがな'}
         self.writer._write_final_result()
-        mock_zip_file.write.assert_called_with('tmp', 'ひらがな.csv')
+        mock_zip_file.write.assert_called_with(
+            'tmp',
+            os.path.join(self.writer.archive_basepath, 'ひらがな.csv')
+        )
 
     def test_zipped_export_writer_utf8(self):
         mock_zip_file = self.MockZipFile.return_value
         self.writer.table_names = {0: '\xe3\x81\xb2\xe3\x82\x89\xe3\x81\x8c\xe3\x81\xaa'}
         self.writer._write_final_result()
-        mock_zip_file.write.assert_called_with('tmp', 'ひらがな.csv')
+        mock_zip_file.write.assert_called_with(
+            'tmp',
+            os.path.join(self.writer.archive_basepath, 'ひらがな.csv')
+        )
 
 
 class CsvFileWriterTests(SimpleTestCase):

--- a/corehq/ex-submodules/couchexport/writers.py
+++ b/corehq/ex-submodules/couchexport/writers.py
@@ -297,7 +297,7 @@ class ZippedExportWriter(OnDiskExportWriter):
         self.file.seek(0)
 
     def _get_archive_filename(self, name):
-        return os.path.join(self.archive_basepath, u'{}{}'.format(name, self.table_file_extension))
+        return os.path.join(self.archive_basepath, '{}{}'.format(name, self.table_file_extension))
 
 
 class CsvExportWriter(ZippedExportWriter):

